contrib/ci/configure.sh

```diff
@@ -81,6 +81,12 @@ if [[ "$DISTRO_BRANCH" == -redhat-fedora-3[5-9]* ||
     )
 fi
 
+if [[ "$DISTRO_BRANCH" == -redhat-fedora-* ]]; then
+    CONFIGURE_ARG_LIST+=(
+        "--with-fido2"
```

Comment:
Can it also be added to `fedconfig` function in contrib/fedora/bashrc_sssd ?

Reply:
Done